components/widgets/team_icon/team_icon.tsx

```diff
@@ -41,6 +41,8 @@ export class TeamIcon extends React.PureComponent<Props> {
             throw new Error('Either `url` or `name` prop is required ');
         }
         const hoverCss = withHover ? '' : 'no-hover';
+
+        // FIXME Nowhere does imageURLForTeam seem to check for display_name.
```

Comment:
Do you have more details on this so we could file a ticket?

Reply:
Sure in

 ```export function imageURLForTeam(teamIdOrObject) {
    return teamIdOrObject.last_team_icon_update ? Client4.getTeamIconUrl(teamIdOrObject.id, teamIdOrObject.last_team_icon_update) : null;
}```

I don't see the usage of display_name.